### DISCUSSION TITLE
Product subscriptions

### DISCUSSION THREAD

**Discussion by pjtf93** [saleor/saleor#9359] [Q&A]
Hello everyone, i need to implement a subscriptions system on my store and i looking for the best way to do this, i currently use Stripe as my payment gateway and i know they support subscriptions but i'm not sure how this would work together with Saleor. Is there an example available for this? or what would you recommend for this case.

Thanks

**Comment by karolkielecki** [comment]
Hi @pjtf93, Saleor doesn't support subscriptions. Afaik, there is no such example available. However, you can build an app using Stripe and metadata. Learn more on how to build an app here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts.
Good luck!
